packages/sdk/src/channel-manager/channel.ts

```diff
@@ -927,34 +939,6 @@ export async function estimateChannelEditCommentFee({
   });
 }
 
-/**
- * All read contract types needed for estimating fee for posting a comment to a channel
- */
-export type EstimateChannelPostCommentFeeReadContractType =
```

Comment:
since we just released this yesterday, i doubt any one will be using them, so should be fine to be removed